sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapter.java

```diff
@@ -326,18 +329,22 @@ private Optional<SclReportItem> updateSourceDataSetsAndControlBlocks(TExtRef ext
         return Optional.empty();
     }
 
-    private void createDataSetWithFCDA(TExtRef extRef, LDeviceAdapter sourceLDevice, ResumedDataTemplate sourceDa, String datasetSuffix) {
-        DataSetAdapter dataSetAdapter = sourceLDevice.getLN0Adapter().createDataSetIfNotExists("DS_" + datasetSuffix, ControlBlockEnum.from(extRef.getServiceType()));
+    private void setExtRefSrcAttributes(TExtRef extRef, String cbName) {
```

Comment:
setExtRefSrcCBNameAndLdInst

Reply:
Actually, it also concerns : `srcPrefix`, `srcLNInst` and `srcLNClass` which are set to null because associated ControlBlock is in LN0 (this part of the issue requirements). I added these 3 attributes to be more  explicit.